candidate/server.py

```
@@ -1,23 +1,53 @@
+            content = await resume_file.read()
+            tmp.write(content)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Не удалось сохранить файл: {e}")
+
+    # 2) Извлекаем данные из резюме
+    try:
+        resume_dict = extract_brief(temp_path)
+    except Exception as e:
+        # удаляем временный файл перед поднятием ошибки
+        os.unlink(temp_path)
+        raise HTTPException(status_code=400, detail=f"Ошибка при обработке резюме: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)